src/components/10-atoms/input-text/README.md

```diff
@@ -73,10 +74,27 @@ The string-valued `label` provides the label text as HTML.
 
 The string-valued `refId` sets the reference for label and input, if no `refId` is set a random id number will be created.
 
-### name\*
+### name (required within `<form>` tag)
 
 The string-valued `name` of the element for purposes of form submission.
 
+### maxLength
+
+Defines a maximum number of characters, that a user has available. Attention: If the user uses ALL available characters, a validation message will occur and the form should not be submittable. Therefore, the counter of available characters left displays always the value of `maxLength -1`. The reason for this strange behavior is, that another implementation is not possible without getting rid of the internal html `<textfield>` element, which would degrade the accessibility rating of the component.
```

Comment:
Your wording is problematic as it stands: the form will be submittable anyway as currently implemented. i suggested rewriting this paragraph.